features/step_definitions/brief_response_steps.rb

```diff
@@ -33,8 +33,8 @@
   submit_supplier_declaration(@framework['slug'], @supplier["id"], {})
 end
 
-Given /^that supplier has a service on the (.*) lot$/ do |lot_slug|
```

Comment:
~~Feels like this version without a role should still be useful to us? e.g. for outcomes, labs, cloud-hosting, whatever?~~

Ah OK, just spotted it will still work without any role specified.